x-pack/plugins/ml/public/url_generator.ts

```diff
@@ -24,11 +26,81 @@ declare module '../../../../src/plugins/share/public' {
 
 export const ML_APP_URL_GENERATOR = 'ML_APP_URL_GENERATOR';
 
+export const ML_PAGES = {
+  ANOMALY_DETECTION: 'jobs',
```

Comment:
Maybe change to `ANOMALY_DETECTION_JOBS_MANAGE`?

Reply:
Updated here https://github.com/elastic/kibana/pull/75696/commits/60d6799ffea948f9f31187805fd40729c2088544